Sources/StreamChat/Database/DatabaseContainer.swift

```diff
@@ -174,7 +173,9 @@ class DatabaseContainer: NSPersistentContainer {
 
         FetchCache.clear()
 
-        setupLoggerForDatabaseChanges()
+        if LogConfig.level == .debug {
```

Comment:
was this causing some issues?

Reply:
No, I should take it out. Kind of pointless to observe in non-debug modes because it forwards the logging to log.debug